cypress/helpers/cypressTasks/dynamo/dynamo-helpers.ts

```diff
@@ -51,6 +51,49 @@ export const expireUserConfirmationCode = async (
   return null;
 };
 
+export const toggleFeatureFlag = async ({
+  flag,
+  flagValue,
+}: {
+  flag: string;
+  flagValue: any;
```

Comment:
Aren't our feature flag values booleans?

Reply:
> Aren't our feature flag values booleans?

I believe the Clerk of Court and Chief Judge values are formatted like feature flags (that is, their value is stored in the `current` field).